### ISSUE TITLE
San d'Oria Quest "The Vicasque's Sermon" onTrade Script Error

### ISSUE THREAD

**Issue by eyes-and-brain** [topaz-next/topaz#2364]
**_I have:_**

- [x] searched existing issues (https://github.com/topaz-next/topaz/issues) to see if the issue has already been opened
- [x] checked the commit log to see if the issue has been resolved since my server was last updated

**_Additional Information_** (Steps to reproduce/Expected behavior) **:** 

https://github.com/topaz-next/topaz/blob/4cd4cac9b8e53bd7c8d129009bdfe5201dd8ccb2/scripts/zones/Northern_San_dOria/npcs/Abioleget.lua#L14-L23

I think the variable "sermonQuest" is undefined. (L15)

```
    local sermonQuest = player:getQuestStatus(SANDORIA, tpz.quest.id.sandoria.THE_VICASQUE_S_SERMON)
```

**Comment by wrenffxi** [comment]
fixed in sol_refactor. will close issue once it lands in release.

**Comment by wrenffxi** [comment]
closed via #2376 merging into release
